tests/queries/saved_queries/api_tests.py

```diff
@@ -745,3 +751,48 @@ def test_export_not_allowed(self):
         uri = f"api/v1/saved_query/export/?q={prison.dumps(argument)}"
         rv = self.client.get(uri)
         assert rv.status_code == 404
+
+    def create_saved_query_import(self):
+        buf = BytesIO()
+        with ZipFile(buf, "w") as bundle:
+            with bundle.open("saved_query_export/metadata.yaml", "w") as fp:
+                fp.write(yaml.safe_dump(saved_queries_metadata_config).encode())
+            with bundle.open(
+                "saved_query_export/databases/imported_database.yaml", "w"
+            ) as fp:
+                fp.write(yaml.safe_dump(database_config).encode())
+            with bundle.open("saved_query_export/queries/imported_database/public/imported_saved_query.yaml", "w") as fp:
+                fp.write(yaml.safe_dump(saved_queries_config).encode())
+        buf.seek(0)
+        return buf
+
+    @pytest.mark.usefixtures("create_saved_queries")
```

Comment:
A [fixture](https://docs.pytest.org/en/stable/fixture.html) runs setup code before a test function, and cleanup code after. There's no fixture called `create_saved_queries`, so you should remove this.

Reply:
that makes sense, sort of like a beforeEach afterEach in RTL. Would it be more thorough to add this fixture?